src/CMakeLists.txt

```diff
@@ -462,6 +466,10 @@ ADD_SUBDIRECTORY ( Commands )
 ADD_SUBDIRECTORY ( Desktop )
 ADD_SUBDIRECTORY ( CommandLine )
 ADD_SUBDIRECTORY ( Tests )
+if (CMAKE_COMPILER_IS_GNUCC)
```

Comment:
The dispatcher won't run any function that isn't supposed to work. The way I set it up now it doesn't even know about dot_avx() etc. in the non-x86_64 case (https://github.com/kloewe/mod-dot/commit/e226be9). But won't the compilation of dot_avx.c fail because of unsuitable flags like -mavx?